distribution/changelog.txt

```diff
@@ -4,6 +4,7 @@
 - Fix: [#20907] RCT1/AA scenarios use the 4-across train for the Inverted Roller Coaster.
 - Fix: [#21332] Mini Helicopters and Monorail Cycles ride types are swapped in research within RCT1 scenarios.
 - Fix: [#21347] Too many options are hidden if the platform has no file picker.
+- Fix: [#21317] Track designer allow proceeding without an object selected.
```

Comment:
```suggestion
- Fix: [#21317] Track designer allows proceeding without an object selected.
```

Missed an 's' on my edit of the title. Also note that the entry needs to be sorted by issue number (aside from fix type), so you need to move this one a few lines above